software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -339,7 +340,6 @@ protected WinRmTool newWinRmTool(Map<?,?> props) {
             args.putAll(props);
             args.configure(SshTool.PROP_HOST, getAddress().getHostAddress());
             args.configure(WinRmTool.USE_NTLM, getConfig(WinRmMachineLocation.USE_NTLM));
-            args.configure(WinRmTool.USE_HTTPS_WINRM, getConfig(WinRmMachineLocation.USE_HTTPS_WINRM));
```

Comment:
Yes it works. Check https://github.com/apache/brooklyn-server/blob/master/software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java#L153-L157 and the loop above.